app/grandchallenge/components/models.py

```diff
@@ -1171,6 +1171,15 @@ def component_interface_value_path(instance, filename):
     )
 
 
+class ComponentInterfaceValueManager(models.Manager):
+
+    def get_first_or_create(self, **kwargs):
+        try:
+            return self.get_or_create(**kwargs)
+        except MultipleObjectsReturned:
+            return self.filter(**kwargs).first(), False
```

Comment:
Well, we have a lot of duplicate CIVs in our database. That's because of how we used to create CIVs. We used to just create a new CIV everything we create a job (and a year ago also every time we update or create a display set or archive item). I don't think getting alerted helps us here, since we already know that this is the case, right? In terms of fixing, we might want to delete orphaned CIVs (that no longer belong to a display set, archive item or job), but that will probably still leave us with some duplicates that we cannot delete.